Scripts/prepare_release.py

```diff
@@ -87,56 +114,58 @@ def render(self) -> str:
             if ext not in self.fragments:
                 continue
             rendered.append(f"## {title}")
-            for pth in self.fragments[ext]:
-                text = pth.read_text(encoding="utf-8")
-                link = f"(https://github.com/dafny-lang/dafny/pull/{pth.name})"
-                entry = indent(f"- {text}\n{link}", "  ").lstrip()
+            for fr in self.fragments[ext]:
+                link = f"(https://github.com/dafny-lang/dafny/pull/{fr.pr})"
+                entry = indent(f"- {fr.contents}\n{link}", "  ").lstrip()
```

Comment:
I don't think you want to add a newline between the contents and the link. At least in GitHub, where the release nodes can be seen as a formatted markdown file, even though the link seems long, it will actually look like this: #1239